package.json

```diff
@@ -26,6 +26,12 @@
   },
   "author": "",
   "license": "Apache-2.0",
+  "peerDependencies": {
+    "lodash": "^4.17.5",
+    "radium": "^0.25.2",
+    "react": "~15.4.0",
+    "react-dom": "~15.4.0"
```

Comment:
Heads up @jmkulwik that if you bump the react version to 15.6 in our main repo, that you may need to bump these too.